package io.github.kappa243.remitly2025.exceptions;

public class ChildBranchesFoundException extends Exception {
}
